README.md

```diff
@@ -179,6 +192,6 @@ If you like this project, don't forget to star.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](MIT.md) licensed.
```

Comment:
- [ ] The MIT link leads to a 404 page. Kindly ensure that you implemented it well as advised by the previous reviewer. 

![image](https://github.com/sibomana123/MV_Hello__project/assets/84156714/f20a0c4a-60a8-4e0f-9ce6-892b58c6bbd9)